test/test_c_stuff_sack.c

```diff
@@ -0,0 +1,259 @@
+#include <stdbool.h>
+#include <stdint.h>
+#include <string.h>
+#include <stdio.h>
+
+#include "external/unity/src/unity.h"
```

Comment:
You should #include external stuff with <> instead of "". See my comment in the BUILD file for unity.